PRIVACY.md

```diff
@@ -1,22 +1,53 @@
-# Semgrep Privacy Policy
+# Semgrep CI Privacy Policy
 
-Semgrep collects usage data to provide useful results and to help us improve the product. We send two types of data to r2c servers: scan data and findings data.
+## Metrics
 
-## Scan data
+Semgrep CI (also known as semgrep-action or semgrep-agent) may collect non-identifiable aggregate metrics to help improve the product. This document describes:
+
+- the principles that guide our data-collection decisions
+- the breakdown of the data that are and are not collected
+- how to opt-out of Semgrep CI’s metrics
+
+## Principles
+
+These principles inform our decisions around data collection:
+
+1. *Transparency*: Collect and use data in a way that is clearly explained to the user and benefits them
+2. *User control*: Put users in control of their data at all times
+3. *Limited data*: Collect what is needed, de-identify where possible, and delete when no longer necessary
```

Comment:
```suggestion
1. **Transparency**: Collect and use data in a way that is clearly explained to the user and benefits them
2. **User control**: Put users in control of their data at all times
3. **Limited data**: Collect what is needed, de-identify where possible, and delete when no longer necessary
```

Adding bold for consistency with styling on https://github.com/returntocorp/semgrep/blob/develop/PRIVACY.md